apps/percona/deployers/percona.yml

```diff
@@ -34,6 +34,12 @@ spec:
           volumeMounts:
             - mountPath: /var/lib/mysql
               name: data-vol
+          livenessProbe:
```

Comment:
we may need to set `restartPolicy` as never (or) the checks that we in litmus job for AUT liveness need to have tasks for looking for any restarts.

Reply:
The checks done for AUT liveness in litmus chaos jobs considers temporary loss of app endpoint if there are restarts - and has retries with specific intervals. So it will continue to be an indicator of app health. So, in case of spurious restarts by the livenessProbe (due to say, db init is delayed, etc.,) if it can recover and continue to write - it should still be OK. In case of genuine probe failures (such as RO or DB corruption), the restarts will anyways result in app staying in crashLoopBackoff - which is desirable for us -- and in this case the external liveness to will fails as the retry checks will be exhausted

```